explorer/templates.go

```diff
@@ -158,9 +158,15 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 		"add": func(a, b int64) int64 {
 			return a + b
 		},
+		"intAdd": func(a, b int) int {
```

Comment:
Note that there is already a built-in `add` function.  Unfortunately the "add" above masked the built-in, so to keep with the `int64` versions being the simplest names, I acknowledge `intAdd` is necessary now.  Just please take note.